proposals/simd/NewOpcodes.md

```diff
@@ -0,0 +1,124 @@
+| Memory instruction | opcode |
+| ------------------ | ------ |
+| v128.load          | 0x00   |
+| i16x8.load8x8_s    | 0x01   |
+| i16x8.load8x8_u    | 0x02   |
+| i32x4.load16x4_s   | 0x03   |
+| i32x4.load16x4_u   | 0x04   |
+| i64x2.load32x2_s   | 0x05   |
+| i64x2.load32x2_u   | 0x06   |
+| v8x16.load_splat   | 0x07   |
+| v16x8.load_splat   | 0x08   |
+| v32x4.load_splat   | 0x09   |
+| v64x2.load_splat   | 0x0a   |
+| v128.store         | 0x0b   |
+
+| Basic instruction    | opcode |
+| -------------------- | ------ |
+| v128.const           | 0x0c   |
+| v8x16.shuffle        | 0x0d   |
+| v8x16.swizzle        | 0x0e   |
+| i8x16.splat          | 0x0f   |
```

Comment:
The ordering here is mildly unsatisfying for decoding purposes. Similarly to the existing spec, this interleaves splat/extract/replace lane, and they have different encoding formats. Thoughts on grouping these by instruction type instead? (splats followed by lane operations)

Reply:
Yeah I think that sounds reasonable 👍